src/main/java/net/webstructor/cat/HtmlStripper.java

```diff
@@ -98,7 +103,7 @@ public static String stripHtmlAnchor(String url){
          */
         public static String attribute(String source, int pos, String name, String baseHref){
 //TODO: lowercase
-        	int till = 1024;

```

Comment:
While fixing formatting is a good thing, too many formatting fixes may get PR more difficult to review...

Reply:
I understand. Sorry about that. Most of them were not intentional. I should've filtered them out before the commit. Will clean up on following commits.